servers/zms/src/main/java/com/yahoo/athenz/zms/store/file/FileConnection.java

```diff
@@ -405,6 +405,29 @@ public boolean insertDomainTemplate(String domainName, String templateName, Stri
         return true;
     }
 
+    @Override
+    public boolean updateDomainTemplate(String domainName, String templateName, TemplateMetaData templateMetaData) {
```

Comment:
this implementation doesn't look correct to me. we're setting a templateMetaList object which contains only a single templateMeta which only has a version number. where are the rest of the templates or where are the template names we're storing?

Reply:
done.